src/prefect/client/orion.py

```diff
@@ -162,7 +162,7 @@ def __init__(
         )
 
         # See https://www.python-httpx.org/advanced/#custom-transports
-        # 
+        #
```

Comment:
@chrisguidry :P

Reply:
The _one_ thing I quickly tweaked in the GH UI hahaha